smtp_service_test.go

```diff
@@ -17,26 +17,20 @@ func (suite *SendpulseTestSuite) TestSmtpService_Send() {
 		}`)
 	})
 
+	userSlice := make([]User, 1)
+	userSlice[0] = User{Name: "Andy", Email: "Forest"}
```

Comment:
I don't think that `userSlice` it's the right variable name because the type is present in the name. It also makes sense to initialize in one line. So it's better to do that:
`users := []User{{Name: "Andy", Email: "Forest"},}`